docs/monitoring/05-01-alertmanager.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Alertmanager sub-chart
+type: Configuration
+---
+
+To configure the Alertmanager sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents:
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
```

Comment:
that configuration options should be put into a secret, not a configmap as they contain secrets. That needs to be mentioned somehow